docs/guides/monitor/lamp-stack.md

```diff
@@ -0,0 +1,227 @@
+collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/apache).
+
+Actually, there's nothing for you to do to enable Apache monitoring with Netdata.
+
+Apache comes with `mod_status` enabled by default these days, and Netdata is smart enough to look for metrics at that
+endpoint without you configuring it. Netdata is already collecting [`mod_status`
+metrics](https://httpd.apache.org/docs/2.4/mod/mod_status.html) _and_ additional metrics from parsing Apache's
+`access.log` file, which is everything you need for Apache monitoring.
+
+## Enable MySQL monitoring
+
+Because your MySQL database is password-protected, you do need to tell MySQL to allow the `netdata` user to connect to
+without a password. Netdata's [MySQL data
+collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/mysql) collects metrics in _read-only_
+mode, without being able to alter or affect operations in any way.
+
+First, log into the MySQL shell. Then, run the following three commands, one at a time:
+
+```mysql
+CREATE USER 'netdata'@'localhost';
+GRANT USAGE, REPLICATION CLIENT, PROCESS ON *.* TO 'netdata'@'localhost';
+FLUSH PRIVILEGES;
+```
+
+Run `sudo systemctl restart netdata`, or the [appropriate alternative for your
+system](/docs/configure/start-stop-restart.md), to collect dozens of metrics every second for robust MySQL monitoring.
+
+## Enable PHP monitoring
+
+Unlike Apache or MySQL, PHP isn't a service that you can monitor directly, unless you instrument a PHP-based application
+with [StatsD](/collectors/statsd.plugin/README.md).
+
+However, if you use [PHP-FPM](https://php-fpm.org/) in your LAMP stack, you can monitor that process with our [PHP-FPM
+data collector](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/phpfpm).
+
+Open your PHP-FPM configuration for editing, replacing `7.4` with your version of PHP:
+
+```bash
+sudo nano /etc/php/7.4/fpm/pool.d/www.conf
+```
+
+> Not sure what version of PHP you're using? Run `php -v`.
+
+Find the line that reads `;pm.status_path = /status` and remove the `;` so it looks like this:
+
+```conf
+pm.status_path = /status
+```
+
+Next, add a new `/status` endpoint to Apache. Open the Apache configuration file you're using for your LAMP stack.
+
+```bash
+sudo nano /etc/apache2/sites-available/your_lamp_stack.conf
+```
+
+Add the following to the end of the file, again replacing `7.4` with your version of PHP:
+
+```apache
+ProxyPass "/status" "unix:/run/php/php7.4-fpm.sock|fcgi://localhost"
+```
+
+Save and close the file. Finally, restart the PHP-FPM, Apache, and Netdata processes.
+
+```bash
+sudo systemctl restart php7.4-fpm.service
+sudo systemctl restart apache2
+sudo systemctl restart netdata
+```
+
+As the Netdata Agent starts up again, it automatically connects to the new `127.0.0.1/status` page and collects
+per-second PHP-FPM metrics to get you started with PHP monitoring.
+
```

Comment:
having setup netdata for a LAMP stack with a sysadmin (https://odyslam.com/blog/Netdata-installation-production-web-server), the sysadmin noted that a very useful collector is the httpcheck collector. From the blog: 
> This is handful in case the PHP server has crashed, so nginx does not return the proper website, but rather the nginx home screen

He mentioned that http-check + regex on the website content is most useful to ensure that it's up and running. 

perhaps we can add a note without adding instructions

Reply:
Added a note about this collector! The whole regex idea would make for a great standalone guide... 🤔